src/main/java/Main.java

```diff
@@ -0,0 +1,60 @@
+import java.lang.reflect.Array;
+import java.util.ArrayList;
+import java.util.HashMap;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        HashMap<String, String> ownersAnimals= new HashMap<>();
```

Comment:
You should do it in this way: Map<String, String> ownersAnimals= new HashMap<>();
Interface Map is more general then implementatnion like HashMap